core/server/data/xml/sitemap/base-generator.js

```diff
@@ -65,10 +64,6 @@ class BaseSiteMapGenerator {
         this.lastModified = Date.now();
     }
 
-    getPriorityForDatum() {
```

Comment:
There are some more occurrences of `getPriorityForDatum` if you do search e.g. in tag-generator.